### DISCUSSION TITLE
platform-cache: TTL units – s or ms? (cache-manager is ms now)

### DISCUSSION THREAD

**Discussion by mikkokam** [TypedProject/ts-express-decorators#2331] [Q&A] [answered]
Hi,
seems like the docs indicate the TTL is in **seconds**. Also, the values indicate this to be the case.

However, cache-manager switched to **ms in 5.0.0**?

[https://github.com/node-cache-manager/node-cache-manager/issues/348](https://github.com/node-cache-manager/node-cache-manager/issues/348)

Ts.ED platform-cache seems to use the new one.
`"cache-manager": "^5.1.3",`

----

An example from the docs:

[https://tsed.io/docs/cache.html#refresh-cache-keys-in-background](https://tsed.io/docs/cache.html#refresh-cache-keys-in-background) 

`@UseCache({ttl: 3600, refreshThreshold: 900})`

> In this example, the configured ttl is 1 hour and the threshold is 15 minutes.

**Comment by Romakita** [comment, marked as answer]
The documentation is correct. To avoid breaking change, Ts.ED convert TTL to `ms`

**Reply by mikkokam** [reply]
Thanks.
